src/Components/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import './MovieCard.css'
+
+const MovieCard = ({ info }) => {
+  const releaseYear = info.release_date.split('-')[0]
+  const rating = info.average_rating.toFixed(1)
+
+  return (
+    <article className='movie-card'>
+      <img
+        className='poster'
+        src={info.poster_path}
+        alt={`${info.title} movie poster`}
+      />
+      <div className="movie-specs">
+        <h2 className='movie-title'>
+          {info.title}
+        </h2>
+        <p className='movie-release'>
+          {releaseYear}
+        </p>
+        <p className='movie-rating'>
+          {`Tater Rating: ${rating}`}
+        </p>
+      </div>
+    </article>
+  )
+}
```

Comment:
I think these cards look great! Smart to modify the data to store in variables to keep things cleaner, and helpful to have the separate text elements grouped for layout manipulations/styling.